examples/beam/without_job_server/beam_wordcount_java.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: beam-wordcount-java
+spec:
+  template:
+    metadata:
+      labels:
+        app: beam-wordcount-java
+    spec:
+      containers:
+        - name: beam-wordcount-java
+          image: apache/beam_java_sdk:2.23.0
+          command: ["/bin/bash", "-c"]
+          args:
+          - |
+            apt-get update && apt-get install -y maven
+            mvn archetype:generate -DarchetypeGroupId=org.apache.beam -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-examples -DarchetypeVersion=2.23.0 -DgroupId=org.example -DartifactId=word-count-beam -Dversion="0.1" -Dpackage=org.apache.beam.examples -DinteractiveMode=false
+            cd word-count-beam
+            mvn package -Pflink-runner
+            mvn exec:java -Dexec.mainClass=org.apache.beam.examples.WordCount \
+                -Pflink-runner \
+                -Dexec.args="--runner=FlinkRunner \
+                  --inputFile=pom.xml \
+                  --output=. \
```

Comment:
The current dir seems to have permission issue in my test, I replaced it with `/tmp/output` which is the same as that of the py version.